### ISSUE TITLE
Move from Sass to Post CSS

### ISSUE THREAD

**Issue by aminalhazwani** [bwinton/StyleGuide#28]
Since Tachyons is written in Post CSS we could move also the custom styles from pre to postprocessing

**Comment by bwinton** [comment]
This seems like a large change, and I'm not really sure what the benefit is…  Can you comment a little more on why we would want to do this?

**Comment by bwinton** [comment]
Okay…  and I guess Tachyons is something we want.  Why do we want to include it as an npm module, though?

**Comment by bwinton** [comment]
(Also, apparently they maintain a SASS version [over here](https://github.com/tachyons-css/tachyons-sass), so we could just use that…)

**Comment by aminalhazwani** [comment]
Because as npm we could customize the source code and I don't have to create new classes that overwrite the current ones. So for example if for the purpose of the website we need different breakpoints we could easily update them without having to create new overwriting rules. What do you think?

**Comment by bwinton** [comment]
I'm not sure how we would customize their source code with npm.  Like, you could make customizations, but they wouldn't appear in my repo, because when I ran `npm install` it would get the official version, not your customized one…  But I think I don't understand what you're trying to do.

**Comment by aminalhazwani** [comment]
One of the things I really appreciate of CSS is its simplicity. Pre-processors were a great thing, they actually brought a lot of improvements to vanilla CSS (like jQuery did for Javascript) but today they look more like scripting and less like styling. And you usually tend to over complicate your styles with mixins and functions (at least I did this mistake several times 😁 ). Post CSS brings you back to vanilla CSS, so that you are more aware of the compiled code, but without losing the fun of great features from level 4 and level 5 CSS.

**Comment by aminalhazwani** [comment]
If we include it as a dev dependency cannot we actually modify the source code, compile it and use it in the website? I may be completely wrong though.

**Comment by bwinton** [comment]
No, that's not how devDependencies work.  They're just stuff you would need to build the site.  You can't make modifications to them, and have those modifications spread to other forks of the code.

**Comment by aminalhazwani** [comment]
Sorry, my bad! Is there a way to accomplish what I just mentioned? I still see the benefit of including it as npm though. It would be one less external call that the website has to made 😄

**Comment by bwinton** [comment]
Not that I can think of, but I'll look into it and let you know for sure.
